src/OrchardCore.Modules/OrchardCore.Media.Azure/Services/MediaBlobStorageOptionsConfiguration.cs

```diff
@@ -22,4 +22,10 @@ public MediaBlobStorageOptionsConfiguration(
     protected override MediaBlobStorageOptions GetRawOptions()
         => _shellConfiguration.GetSection("OrchardCore_Media_Azure")
         .Get<MediaBlobStorageOptions>();
+
+    protected override void FurtherConfigure(MediaBlobStorageOptions rawOptions, MediaBlobStorageOptions options)
+    {
+        options.RemoveContainer = rawOptions.RemoveContainer;
+        options.RemoveFilesFromBasePath = rawOptions.RemoveFilesFromBasePath;
+    }
```

Comment:
`RemoveContainer` was not reflecting the value set in appsettings, this addition fixes that.